modules/examples/eureka/requirements-dev.in

```diff
@@ -0,0 +1,13 @@
+awscli
```

Comment:
Are these needed?

Reply:
Per discussion offline, we will remove this file